src/node_crypto.cc

```diff
@@ -818,7 +835,7 @@ void SecureContext::SetOptions(const FunctionCallbackInfo<Value>& args) {
   SecureContext* sc = Unwrap<SecureContext>(args.Holder());
 
   if (args.Length() != 1 || !args[0]->IntegerValue()) {
-    return sc->env()->ThrowTypeError("Bad parameter");
+    return sc->env()->ThrowTypeError("Options must be an integer value");
```

Comment:
@YuriSolovyov Ya, that is why I left it as the function name hints. I don't have a strong opinion about this name anyway.